File: Programmers/Python/Level1/Number_of_p_and_y_in_string.py
def solution(s):
    answer = True
    l = len(s)
    p = 0
    y = 0
    
    for i in range(l):
        if s[i] == 'p' or s[i] == 'P':
            p += 1
        if s[i] == 'y' or s[i] == 'Y':
            y += 1
            
    if y != p:
        return False
            
    return True
    

def solution(s):
    answer = True
    
    # lower() 함수는 문자열 전체를 소문자로 변환
    # count() 함수는 문자열의 개수를 반환
    # ( count() 함수의 매개변수(문자)를 넣어주면 문자의 개수를 반환 )
    
    return s.lower().count('p') == s.lower().count('y')

File: Programmers/C++/Level1/Number_of_p_and_y_in_string.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    int len = s.length();
    int p = 0;
    int y = 0;
    
    for(int i = 0; i < len; i++)
    {
        if(s[i] == 'p' || s[i] == 'P') p++;
        if(s[i] == 'y' || s[i] == 'Y') y++;
    }

    return p == y;
}